### PR TITLE
rename api-checkpointer to pod-checkpointer

### PR DESCRIPTION
**Pull request by aaronlevy**

### CODE REVIEW

**Comment by aaronlevy** [comment]
coreosbot test checkpoint code

**Comment by aaronlevy** [comment]
coreosbot test checkpoint code

File: cmd/checkpoint/main.go [hunk 1]
```diff
@@ -42,6 +43,8 @@ var (
 )
 
 func main() {
+	flag.Set("logtostderr", "true")
```
Comment (xiang90) [inline cmd/checkpoint/main.go]: @pbx0 This is why you did not find any checkpoint logging.

I spent quite a while to figure out why a few days ago... glog is crazy.

Comment (aaronlevy) [inline cmd/checkpoint/main.go]: Yeah. There are also some other issues that logging surfaced. I'll put this in a separate PR

**Comment by xiang90** [comment]
lgtm

**Comment by aaronlevy** [comment]
Can't get it to work for testing yet, so I'll move the logging to diff PR and try and figure out what's going on here.

**Comment by aaronlevy** [comment]
closed in favor of:
https://github.com/kubernetes-incubator/bootkube/pull/227
https://github.com/kubernetes-incubator/bootkube/pull/233
https://github.com/kubernetes-incubator/bootkube/pull/234

**Pull request closed without merging**